src/doc/src/reference/workspaces.md

```diff
@@ -101,6 +102,15 @@ workspace. This can be useful if some path dependencies aren't desired to be
 in the workspace at all, or using a glob pattern and you want to remove a
 directory.
 
+The [`resolver`] key effects how a package that is depended on in multiple
+ways with different features are resolved. Some libraries depend on it being
+set to 2.
+
+An empty `[workspace]` table can be used with a `[package]` to conveniently
+create a workspace with the package and all of its path dependencies.
```

Comment:
This doesn't quite fit within the current section

Maybe we should move this into the "virtual workspace" section and only update that example?

Reply:
I removed this paragraph, seems like this concept is explained well enough in the virtual workspace section

Reply:
The focus of my concern wasn't so much the duplication of information about virtual workspaces but that we are in the section about members and exclude keys and introducing the resolver key as if it belongs in that section.  We then include the resolver key in each further section.  I get the point of including the resolver key in each subsequent section, to try to remind people to  use it, but it doesn't quite fit seeing an unexplained key and one that is only relevant when its a virtual workspace.  This is why I was recommending to make this entire PR only touch the virtual workspace section.

This PR does not have to do all of the lifting for helping users.  We also have https://github.com/rust-lang/cargo/pull/10910.